### ISSUE TITLE
Null values in folder path cause "System.IO.DirectoryNotFoundException: Could not find a part of the path."

### ISSUE THREAD

**Issue by TipsyOne** [rmcrackan/Libation#167]
While using \<series\> and \<series#\> tags in the custom folder path, these values will be null if the book isn't part of a series.  

Instead of ignoring or replacing the null value the software throws an error after the download completes, and doesn't allow you to mark it as finished.  

![libation](https://user-images.githubusercontent.com/95544140/144721954-e91921d0-c88c-42cf-bf90-c38a6aca1066.PNG)

From Settings.json:  
` "FolderTemplate": "<author>\\<title short> - <author> - <series> <series#>",`

From log:  
```
2021-12-04 11:22:45.305 -07:00 [ERR] (at FileManager.FileUtility+<>c__DisplayClass25_0.<SaferMove>b__0()) Failed to move file
System.IO.DirectoryNotFoundException: Could not find a part of the path.
```
[Log202112.log](https://github.com/rmcrackan/Libation/files/7654523/Log202112.log)
[Settings.json.txt](https://github.com/rmcrackan/Libation/files/7654526/Settings.json.txt)

**Comment by rmcrackan** [comment]
Weird. It should omit nulls and blank folders. Does this only happen for you when there's no series present?

**Comment by TipsyOne** [comment]
Yes, it works normally when there's a series name listed.  

To clarify, when the value is null it creates the folder correctly with the null values blanked. This error is only generated at then end when it tries (and fails) to move the decrypted files into the new folder.

**Comment by TipsyOne** [comment]
It looks like it only happens (so far) when the null is at the end of the string and preceded by a space.  

Working:  
`<author>\<series> <series#> - <title short> - <author>`  
`<author>\<title short> - <author><series#>`  

Error:
`<author>\\<title short> - <author> - <series> <series#>`
`<author>\\<title short> - <author> - <series>`
`<author>\\<title short> - <author> - <series#>`

**Comment by rmcrackan** [comment]
Stellar bug reporting. I need more like you at my job. Fixed in version [v6.6.1](https://github.com/rmcrackan/Libation/releases/latest). Thanks again for your help.
